recipe/spyder-menu.json

```diff
@@ -0,0 +1,43 @@
+{
+    "$schema": "https://json-schema.org/draft-07/schema",
+    "$id": "https://schemas.conda.io/menuinst-1.schema.json",
+    "menu_name": "spyder",
+    "menu_items": [
+        {
+            "name": "Spyder",
```

Comment:
Does `menuinst` automatically adds the environment name to the menu item one to differentiate between versions installed in different envs? Or how does that work?

Reply:
`menuinst` does not add any environment distinction. However, `menuinst` does not over-wright existing shortcuts.

To have the application shortcut (bundle) created, `menuinst` must be installed in the _base_ environment. Assuming that a user installs `menuinst` in a base environment (which is not default) and Spyder in two sub-environments, then the application shortcut (bundle) will open the first instance that was installed.

With `menuinst`, it is possible for the user to remove the shortcut (bundle) for one Spyder instance and recreate it for another Spyder instance; uninstalling/reinstalling Spyder is not necessary.

Reply:
Ok, thanks for the clarification @mrclary. Since this is not the goal of this PR, we can see how to address it later.